### ISSUE TITLE
[Bug] ResolvedURL issue with custom activatewindow

### ISSUE THREAD

**Issue by Demonseeed** [jurialmunkey/plugin.video.themoviedb.helper#75]
Hello jurialmonkey! 

I haven’t been able to be as active as I wish, but I hope everything has been well. 

Anyway, I tried to load a movie The Human Centipede (I know, I know) through TMDb Helper, to which an immediate crash of the system. 

Here is a snippet right before the crash in my log:
`WARNING: CreateLoader - unsupported protocol(plugin) in plugin://plugin.video.themoviedb.helper/?info=play&type=movie&tmdb_id=37169
2020-02-03 13:26:07.312 T:139778662049536   ERROR: InputStream: Error opening, plugin://plugin.video.themoviedb.helper/?info=play&type=movie&tmdb_id=37169`

Here is a pastebin for the whole log:
https://pastebin.com/Xp4bLRJX

Thanks again for all the hard work you do.

**Comment by jurialmunkey** [comment]
Is it only this one movie causing a crash?

Log doesn't indicate much other than this message:
```
Quitting due to POSIX signal
```

Which would suggest some type of shutdown signal has been sent from somewhere (e.g. power button on remote).

**Comment by Demonseeed** [comment]
Interesting, good catch on the shutdown. 

I wish it were that simple though. So I checked a couple things. Playing a movie through the addon itself seems to work fine. Playing a movie through a widget seems to work fine as well. 

However, the skin I am using has a custom search very similar to yours in AZ2. Playing every movie I have tried causes the same problem. 

The code from the search list is:
`ActivateWindow(Videos,"plugin://plugin.themoviedb.helper?info=search&type=movie=query=$INFO[Skin.String(CustomSearchTerm)]",return)` (this is directly from the Kodi interface and manually typed. 

It has worked many times in the past, and has no problem loading items in a list when searched. 

Is it possible it’s an issue with the skin and not the addon?

**Comment by Demonseeed** [comment]
To add, I just rolled it back to the known working version (when you added the Discover section, addon version 2.2.5). 

Works perfectly with no changes to search code.

**Comment by jurialmunkey** [comment]
> The code from the search list is:
> `ActivateWindow(Videos,"plugin://plugin.themoviedb.helper?info=search&type=movie=query=$INFO[Skin.String(CustomSearchTerm)]",return)` 

Did you copy paste this command? Because it is wrong - the part `&type=movie=query=` should be `&type=movie&query=` (the `=` between movie and query should be `&`). Also, if this is in skin xml, the `&` needs to be replaced with `&amp;`

The correct command from inside skin xml would be:

```
ActivateWindow(Videos,"plugin://plugin.themoviedb.helper?info=search&amp;type=movie&amp;query=$INFO[Skin.String(CustomSearchTerm)]",return)
```

**Comment by Demonseeed** [comment]
Sorry about the typo! I did not copy and paste it as I am doing this on my phone and Kodi is on my LibreELEC device. It is actually an & symbol and in the xml files are `&amp;`. It also works on previous versions as well as a double-check that the code works. 

I appreciate checking it out and helping me as always👌😄

**Comment by jurialmunkey** [comment]
@Demonseeed - Okay, I'm almost 100% certain I know what the issue is (your rolling back to previous versions to find out when the issue was introduced really helped!!!).

Can you test latest git version and see if it fixes the problem?

**Comment by Demonseeed** [comment]
Man, that was fast! I will definitely check it out, but it will have to wait till tomorrow. 

I’m glad that was able to help, I wanted to give as much information as I could! 😄

**Comment by jurialmunkey** [comment]
> 
> 
> Man, that was fast! I will definitely check it out, but it will have to wait till tomorrow.
> 
> I’m glad that was able to help, I wanted to give as much information as I could! 😄

No worries. I'm fairly certain it is related to this bug I had previously which was introduced around that time: https://github.com/jurialmunkey/plugin.video.themoviedb.helper/issues/60

When I fixed that bug, I didn't consider calling the plugin the way AuraMod does. Plugin thinks it is being called from library as a `.strm` file and so it sets a dummy resolvedurl to prevent this bug: https://github.com/jurialmunkey/plugin.video.themoviedb.helper/issues/52

But that causes the first bug. Latest update I've changed the method to determine if it is playing from a strm file in the library, so hopefully shouldn't have this problem any more.
